backend/app/management/commands/syncdb.py

```diff
@@ -88,12 +88,19 @@ def handle(self, *args, **options):
             values = result.get('values', [])
             databases.append(values)
 
+        # Delete database
         if os.path.exists(settings.DB_PATH):
             print_header('Deleting existing db...')
+            for file in os.listdir(settings.MIGRATIONS_DIR):
```

Comment:
Part of the idea behind having the migration history is that if you want to rollback a change to your database models, the history of files in the migration directory gives you a way to do that. Clobbering the whole directory, as we do here, removes that feature.

Buuuuut... I'm okay with this for now. In early prototyping, we're often making a lot of quick changes to the database schema, and no one's reliant on this being stable. Let's open an issue to _remove_ this feature at the end of the summer, once the models are in a more stable state.